src/test/spec/unified_runner/entity.rs

```diff
@@ -76,7 +76,7 @@ impl ClientEntity {
             if let Some(ignore_command_names) = ignore_command_names {
                 if ignore_command_names
                     .iter()
-                    .any(|name| event.command_name() == name)
+                    .any(|name| event.command_name().eq_ignore_ascii_case(name))
```

Comment:
The straightforward way to implement `observeSensitiveCommands: false` was to add the list of redacted commands to the ignore list; since that list is lowercase, this needed to change to be a case-insensitive comparison.